### DISCUSSION TITLE
failing to build helix from source

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#7779] [Troubleshooting]
![Screenshot_2023-07-29_13:35:58](https://github.com/helix-editor/helix/assets/96831387/65fafecf-147b-46bc-b3f1-82feb46f7acb)

OS: Debian 12 bookworm
CPU: AMD Ryzen 5500U

**Comment by ghost** [comment]
![Screenshot_2023-07-29_14:00:49](https://github.com/helix-editor/helix/assets/96831387/64f583f4-91d1-4180-8e1d-e1e91b6d0d38)

**Comment by ymgyt** [comment]
helix requires that the version of rust be [1.65 or higher](https://github.com/helix-editor/helix/blob/224fd5fa29e43c5b2543ab791de5f46abe1ae8a7/rust-toolchain.toml#L2).
Check your `cargo -V`
